src/DogStatsd.php

```diff
@@ -78,21 +78,27 @@ class DogStatsd
      * host,
      * port,
      * socket_path,
+     * cardinality,
      * datadog_host,
      * global_tags,
      * decimal_precision,
      * metric_prefix,
-     * disable_telemetry
+     * disable_telemetry,
+     * container_id,
+     * origin_detecion
      *
      * @param array{
-     *     host: string,
-     *     port: int,
-     *     socket_path: string,
-     *     datadog_host: string,
-     *     global_tags: string[],
-     *     decimal_precision: int,
-     *     metric_prefix: string,
-     *     disable_telemetry: bool
+     *     host?: string,
+     *     port?: int,
+     *     socket_path?: string,
+     *     cardinality?: "none"|"low"|"orchestrator"|"high",
+     *     datadog_host?: string,
+     *     global_tags?: string[],
```

Comment:
This is actually:

```suggestion
     *     global_tags?: string[]|array<string,string>|array<string,null>,
```

This, if you’d like to be less strict and more concise:

```suggestion
     *     global_tags?: array<string|int,?string>,
```

Reply:
I'll update to this, which should hopefully catch everything!

```
     *     global_tags?: string|string[]|array<string,string>|array<string,null>,
```